### PR TITLE
Laboratory Work #1, Zagovorova Elena - 20FPL3

### PR DESCRIPTION
**Pull request by lenazagg**

### CODE REVIEW

File: pipeline.py [hunk 1]
```diff
@@ -1,6 +1,19 @@
 """
```
Comment (dmitry-uraev) [inline pipeline.py:1]: First of all, you should write your crawler code in scrapper.py.
Secondly, please fix your PR name to match the template. For more information, refer to our group chat.

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]

File: scrapper.py [hunk 2]
```diff
@@ -1,6 +1,20 @@
 """
 Scrapper implementation
 """
+import requests
+from bs4 import BeautifulSoup
+import re
+
+
+def main():
```
Comment (dmitry-uraev) [inline scrapper.py:9]: no, please write code under the if __name__ == "__main__": line

File: scrapper.py [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+    response = requests.get('https://vz.ru/')
+    print(response.status_code)
+
+    with open('index.html', 'w', encoding='utf-8') as f:
+        f.write(response.text)
+
+
+main()
```
Comment (dmitry-uraev) [inline scrapper.py:17]: this should be deleted

File: scrapper.py [hunk 4]
```diff
@@ -26,10 +40,20 @@ class Crawler:
...
     """
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        url = 'https://vz.ru/news/'
```
Comment (dmitry-uraev) [inline scrapper.py:48]: good, but all this should be written under the find_articles function. extract_url extracts specific url only

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
Need comment fixes and lets finish till the next Wednesday

File: scrapper.py (hunk 4 above)
Comment (lenazagg) [reply scrapper.py:45]: I am sorry, something happened with index.html when I pushed changes in scrapper.py, can you please tell me how can I fix it (when I open my pull request, I see "HTML" in languages)?

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
It is time to implement parser. Make sure you have green CI on Friday before the assignment.

**Pull request closed without merging**